src/constants.js

```diff
@@ -1,5 +1,4 @@
-import remote from 'remote';
-const app = remote.require('app');
+const {app} = require('electron').remote;
```

Comment:
Oh cool, we no longer need to do `remote.require`? Can you split this into the following:

```js
import { remote } from 'electron';
const { app } = remote;
```

Just want to standardize on `import` syntax over `require` if we no longer need it.